server/topic.go

```diff
@@ -2580,24 +2586,47 @@ func (t *Topic) mostRecentSession() *Session {
 	return sess
 }
 
-func (t *Topic) suspend() {
-	atomic.StoreInt32((*int32)(&t.suspended), 1)
+const (
+	// Topic is just created.
+	topicStatusNew = 0
+	// Topic is fully initialized.
+	topicStatusLoaded = 1
```

Comment:
I understand `topicStatusLoaded` is only meaningful in ME, P2P and GRP contexts.
FND and SYS topics will forever remain in `topicStatusNew` state.
Is that the intent?
If so, perhaps, SYS and FND should go to `topicStatusLoaded` right away?

Reply:
done